massa-network/src/peer_info_database.rs

```diff
@@ -605,7 +605,7 @@ impl PeerInfoDatabase {
     /// # Argument
     /// * ip : ip address of the considered peer.
     pub fn out_connection_closed(&mut self, ip: &IpAddr) -> Result<(), NetworkError> {
-        let peer = self.peers.get_mut(ip).ok_or_else(|| {
+        let peer = self.peers.get_mut(ip).ok_or({
```

Comment:
sometimes it turns ok_or_else into ok_or, sometimes it does the opposite

Reply:
Any idea why ?

Reply:
> Arguments passed to ok_or are eagerly evaluated; if you are passing the result of a function call, it is recommended to use ok_or_else, which is lazily evaluated.

https://doc.rust-lang.org/std/option/enum.Option.html#method.ok_or